gamestonk_terminal/stocks/stocks_controller.py

```diff
@@ -254,23 +254,27 @@ def call_news(self, other_args: List[str]):
         parser.add_argument(
             "-s",
             "--sources",
-            dest="n_sources",
             default=[],
             nargs="+",
-            help="Show news only from the sources specified (e.g bbc.com yahoo.com)",
+            help="Show news only from the sources specified (e.g bbc yahoo.com)",
         )
 
         try:
             ns_parser = parse_known_args_and_warn(parser, other_args)
             if not ns_parser:
                 return
 
+            sources = ns_parser.sources
```

Comment:
A more elegant solution would be:
```
sources = 
[src for src in ns_parser.sources if ".com" in src] + [f"{src}.com" for src in ns_parser.sources if ".com" not in src]
```

list comprehensions are quite powerful :)